Shortcuts do not take layout into account (1.131)

This issue is probably connected with #1458…

Since updating to 1.131.1, my keyboard shortcuts work as if I were using a QWERTY keyboard (but I have an AZERTY layout).

* `ctrl-z` closes the pane (perceived as `ctrl-w`)
* `ctrl-a` closes the application (perceived as `ctrl-q`)
* etc.

This is pretty unuseable… but solved by downgrading to 1.129.0 or 1.130.0.

## System
```
  Kernel: 6.8.0-100-generic arch: x86_64 bits: 64 compiler: gcc v: 13.3.0 clocksource: tsc
  Desktop: Cinnamon v: 6.6.7 tk: GTK v: 3.24.41 wm: Muffin v: 6.6.3 vt: 7 dm: LightDM v: 1.30.0
    Distro: Linux Mint 22.3 Zena base: Ubuntu 24.04 noble
```

> [@lzbk](https://github.com/lzbk), please [read this comment](https://github.com/pulsar-edit/pulsar/issues/381#issuecomment-2864327897) and follow the instructions. I'm curious to know which values it reports for 1.131.1 and which values it reports for 1.130.0. And if I'm wrong about my assumptions and you're somehow on Wayland (or in some compatibility layer), that would also be useful information.

## 1.130.0
```javascript
require('keyboard-layout').getCurrentKeyboardLayout()
"fr,fr,oss,oss [0]"
require('keyboard-layout').getCurrentKeyboardLanguage()
"fr,fr,oss,oss [0]"
```

## 1.131.1
```javascript
require('keyboard-layout').getCurrentKeyboardLayout()
node:internal/modules/cjs/loader:1154 Uncaught Error: Cannot find module 'keyboard-layout'
Require stack:
- /opt/Pulsar/resources/app.asar/static/index.html
    at Module._resolveFilename (node:internal/modules/cjs/loader:1151:15)
    at a._resolveFilename (node:electron/js2c/renderer_init:2:2669)
    at Function.Module._resolveFilename (/opt/Pulsar/resources/app.asar/src/module-cache.js:317:28)
    at Module._load (node:internal/modules/cjs/loader:992:27)
    at Function.c._load (node:electron/js2c/node_init:2:13672)
    at Function.s._load (node:electron/js2c/renderer_init:2:31018)
    at Module.require (node:internal/modules/cjs/loader:1242:19)
    at require (node:internal/modules/helpers:176:18)
    at <anonymous>:1:1
Module._resolveFilename @ node:internal/modules/cjs/loader:1151
a._resolveFilename @ node:electron/js2c/renderer_init:2
Module._resolveFilename @ /opt/Pulsar/resources/app.asar/src/module-cache.js:317
Module._load @ node:internal/modules/cjs/loader:992
c._load @ node:electron/js2c/node_init:2
s._load @ node:electron/js2c/renderer_init:2
Module.require @ node:internal/modules/cjs/loader:1242
require @ node:internal/modules/helpers:176
(anonymous) @ VM2359:1

require('keyboard-layout').getCurrentKeyboardLanguage()
node:internal/modules/cjs/loader:1154 Uncaught Error: Cannot find module 'keyboard-layout'
Require stack:
- /opt/Pulsar/resources/app.asar/static/index.html
    at Module._resolveFilename (node:internal/modules/cjs/loader:1151:15)
    at a._resolveFilename (node:electron/js2c/renderer_init:2:2669)
    at Function.Module._resolveFilename (/opt/Pulsar/resources/app.asar/src/module-cache.js:317:28)
    at Module._load (node:internal/modules/cjs/loader:992:27)
    at Function.c._load (node:electron/js2c/node_init:2:13672)
    at Function.s._load (node:electron/js2c/renderer_init:2:31018)
    at Module.require (node:internal/modules/cjs/loader:1242:19)
    at require (node:internal/modules/helpers:176:18)
    at <anonymous>:1:1
```

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
     "@babel/core": "7.18.6",
     "@electron/remote": "2.1.2",
     "@formatjs/fast-memoize": "^2.2.6",
-    "@pulsar-edit/atom-keymap": "^9.0.2",
+    "@pulsar-edit/atom-keymap": "^9.0.3",
     "@pulsar-edit/fuzzy-native": "1.3.0",
     "@pulsar-edit/get-scrollbar-style": "^1.0.1",
     "@pulsar-edit/git-utils": "^7.0.1",

